NotToDo/NotToDo/Presentation/AchievementScene/Common/CompositionalLayout.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  CompositionalLayout.swift
+//  NotToDo
+//
+//  Created by JEONGEUN KIM on 2023/01/04.
+//
+
+import UIKit
+
+public class SubItem {
+    static  let spacing: NSCollectionLayoutSpacing = .fixed(10)
+}
+public final class Layout: SubItem {
+    class func compositional_vertical(itemW: NSCollectionLayoutDimension, itemH: NSCollectionLayoutDimension, gWidth: NSCollectionLayoutDimension, gHeight: NSCollectionLayoutDimension, count: Int?) -> NSCollectionLayoutSection {
```

Comment:
```suggestion
class func compositionalVertical(itemW: NSCollectionLayoutDimension, itemH: NSCollectionLayoutDimension, gWidth: NSCollectionLayoutDimension, gHeight: NSCollectionLayoutDimension, count: Int?) -> NSCollectionLayoutSection
```
함수명도 lowerCamelCase로 맞춰주세요 ❗️
추가로 class function으로 선언한 이유가 궁금합니다

Reply:
반복되는 코드를 함수로 빼서,부모 클래스 안에 선언하기 위해서 class 메서드를 사용했습니다!